test/Config.js

```diff
@@ -1,6 +1,6 @@
 import test from 'ava';
 import Config from '../src/Config';
-import { validate } from 'webpack';
+import { validateSchema } from 'webpack';
```

Comment:
I know #46 made this change, however I don't think it's correct. From what I can tell at first glance (and there aren't really any API docs), `validateSchema` expects both schema and options, so if called directly without a schema, it's not validating anything.

Reply:
Looks like you're right!